dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -309,18 +310,15 @@ void NavigationViewItem::OnContentChanged(winrt::IInspectable const& oldContent,
     SuggestedToolTipChanged(newContent);
     UpdateVisualStateNoTransition();
 
-    // Two ways are used to notify the content change on TopNav and asking for a layout update:
-    //  1. The NavigationViewItem can't find its parent NavigationView, just mark it. Possibly NavigationViewItem is moved to overflow but menu is not opened.
-    //  2. NavigationViewItem request update by NavigationView::TopNavigationViewItemContentChanged.
     if (!IsOnLeftNav())
     {
+        // Content has changed for the item, so we want to trigger a re-measure. Set the flag
+        // to indicate that content has changed for this item and force a re-measure if item
+        // has access to NavigationView.
+        m_isContentChangeHandlingDelayedForTopNav = true;
```

Comment:
>m_isContentChangeHandlingDelayedForTopNav [](start = 8, length = 41)

Is this flag even needed anymore, seems like it is always true when this event is raised to NavView